sympy/stats/tests/test_joint_rv.py

```diff
@@ -53,6 +53,32 @@ def test_NormalGamma():
         3*sqrt(10)*gamma(S(7)/4)/(10*sqrt(pi)*gamma(S(5)/4))
     assert marginal_distribution(ng, y)(1) == exp(-S(1)/4)/128
 
+def test_MultivariateBeta():
+    from sympy.stats.joint_rv_types import MultivariateBeta
+    from sympy import gamma
+    a1, a2 = symbols('a1, a2', positive=True)
+    a1_f, a2_f = symbols('a1, a2', positive=False)
+    mb = MultivariateBeta('B', [a1, a2])
+    mb_c = MultivariateBeta('C', a1, a2)
+    assert simplify(density(mb)(1, 2) -
+            S(2)**(a2 - 1)*gamma(a1 + a2)/(gamma(a1)*gamma(a2))) == S(0)
+    assert simplify(marginal_distribution(mb_c, 0)(3) -
+            S(3)**(a1 - 1)*gamma(a1 + a2)/(a2*gamma(a1)*gamma(a2))) == S(0)
+    raises(ValueError, lambda: MultivariateBeta('b1', [a1_f, a2]))
+    raises(ValueError, lambda: MultivariateBeta('b2', [a1, a2_f]))
+    raises(ValueError, lambda: MultivariateBeta('b3', [0, 0]))
+    raises(ValueError, lambda: MultivariateBeta('b4', [a1_f, a2_f]))
+
+def test_MultivariateEwens():
+    from sympy.stats.joint_rv_types import MultivariateEwens
+    n, theta = symbols('n theta', positive = True)
+    theta_f = symbols('t_f', negative = True)
```

Comment:
`negative=True` and similar.

Reply:
Test(s) for symbolic `n` and `theta` with valid assumptions should also be included.

Reply:
@sidhantnagpal  Tests can be added for symbolic `theta` but for `n` it is not possible. The reason for my claim is based on the fact that we can determine `n` from the the length of `syms`. So, one way or the other `n` is an `Integer` and not a `Symbol` and I thought of accepting a fixed `n` from the user rather than figuring out what `n` is from the length of  `syms`.